src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderEclipseStatic.java

```diff
@@ -1,10 +0,0 @@
-//non-compiled with javac: contains specially crafted set of imports for testing
```

Comment:
why removed?

Reply:
I just renamed this file to `InputImportOrderEclipseStatic1.java` and there are 2 more which are using this file so I created two more with the name `InputImportOrderEclipseStatic2.java` and `InputImportOrderEclipseStatic3.java`